front/app/components/ProjectAndFolderCards/ProjectAndFolderCards.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect } from 'react';
```

Comment:
I moved most of `ProjectAndFolderCards/index` into this file. The `index` first has to check how many `adminPublications` there are of each type (active, archived etc) before it can render this component, which uses the `useAdminPublication` hook to request the appropriate `adminPublications`.